includes/wikia/GlobalTitle.php

```diff
@@ -308,7 +322,7 @@ public function getFullURL( $query = '', $variant = false ) {
 			$query = wfArrayToCGI( $query );
 		}
 
-		$url = str_replace( '$1', $namespace . $this->mUrlform, $this->mArticlePath );
+		$url = str_replace( '$1', $namespace . $titleText, $this->mArticlePath );
 		$url = wfAppendQuery( $this->mServer . $url, $query );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$url". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)